ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -2696,13 +2696,15 @@ tests:
   steps:
     cluster_profile: vsphere-dis-2
     env:
+      E2E_RUN_TAGS: '@level0'
       FIPS_ENABLED: "true"
+      TEST_IMPORTANCE: LEVEL0
     leases:
     - env: VSPHERE_BASTION_LEASED_RESOURCE
       resource_type: vsphere-connected-2-quota-slice
     test:
     - chain: openshift-e2e-test-qe-regenerate-cluster-cert
-    - chain: openshift-e2e-test-qe-automated-release
+    - chain: openshift-e2e-test-qe
```

Comment:
This change is unexpected, because we are asked to not run Cypress tests on vsphere profiles.

Reply:
Updated